bfx-pay-woocommerce.php

```diff
@@ -19,14 +19,45 @@
     return;
 }
 
+/**
+ * Register menu elements.
+ */
+class BFX_Admin_Menu
```

Comment:
is there a specific reason why we're using class and not simply a function like other filter functions that we applied?
if not please convert it into a simple function like others